include/crpropa/module/Acceleration.h

```diff
@@ -0,0 +1,170 @@
+#ifndef ACCELERATION_MODULE
+#define ACCELERATION_MODULE
+
+#include <crpropa/Candidate.h>
+#include <crpropa/Module.h>
+#include <crpropa/Vector3.h>
+#include <crpropa/Units.h>
+#include <crpropa/Geometry.h>
+
+#include <string>
+
+namespace crpropa
+{
```

Comment:
Do we have brace placement style agreed? Is it formatted with `clang-format`?

Reply:
I updated the formatting